packages/node_modules/@webex/internal-plugin-ediscovery/src/transforms.js

```diff
@@ -175,34 +175,46 @@ class Transforms {
 
     const promises = [];
 
-    return ctx.webex.internal.ediscovery.getContentSummaryBySpaceId(reportId, activity.targetId)
+    return ctx.webex.internal.ediscovery.getContentContainerByContainerId(reportId, activity.targetId)
       .then((res) => {
-        const space = res.body;
+        const container = res.body;
 
-        if (!space) {
-          const reason = `Space ${activity.targetId} not found - unable to decrypt activity ${activity.activityId}`;
+        if (!container) {
+          const reason = `Container ${activity.targetId} not found - unable to decrypt activity ${activity.activityId}`;
 
           activity.error = reason;
           ctx.webex.logger.error(reason);
 
           return Promise.resolve(object);
         }
 
-        // add space warning properties to the activity - these will be recorded in the downloader
-        if (space.warning) {
-          activity.spaceWarning = space.warning;
+        // add warning properties to the activity - these will be recorded in the downloader
+        if (container.warning) {
+          activity.spaceWarning = container.warning; // Depreciate this property
+          activity.containerWarning = container.warning;
         }
 
         // set space name and participants on activity
-        if (space.spaceName) {
-          activity.spaceName = space.spaceName;
+        if (container.containerName) {
+          activity.spaceName = container.containerName; // Depreciate this property
+          activity.containerName = container.containerName;
         }
-        else if (space.isOneOnOne && space.participantDisplayNames && space.participantDisplayNames.length === 2) {
+        else if (container.isOneOnOne) {
+          let displayNames = '';
+
+          for (const participant of new Set([...container.participants || [], ...container.formerParticipants || []])) {
+            if (displayNames) {
+              displayNames += ' & ';
+            }
+            displayNames += participant.displayName;
```

Comment:
There was a small change in how `participants` are listed in the new API. Previously they were just UUIDs. Now they're participant objects.

Reply:
It strike me that this code would be more appropriate in the client. However I may have argued the opposite in the past as it could duplicate the work if there were multiple clients. Keeping the same approach for now.